pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+<h4> Issue(Describe in 2-3 lines what the current issue or background is) : </h4>
+<br>
+<h4> Fix or Changes done(Describe in 2-3 lines changes or fix made) : </h4>
```

Comment:
nit. Can rename this to "Description" tag which describes the proposed changes of the PR